src/main/java/taxi/controller/GetMyCurrentCarsControlle.java

```diff
@@ -0,0 +1,38 @@
+package taxi.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetMyCurrentCarsControlle extends HttpServlet {
+    private Injector injector;
+    private CarService carService;
+
+    @Override
+    public void init(ServletConfig config) {
+        injector = Injector.getInstance("taxi");
+        carService = (CarService) injector.getInstance(CarService.class);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        HttpSession session = req.getSession();
+        Long driverId = (Long) session.getAttribute("driver_id");
+        List<Car> driverCars = carService.getAllByDriver(driverId);
+        req.setAttribute("cars", driverCars);
+        if (driverId == null) {
+            resp.sendRedirect(req.getContextPath() + "login");
+            return;
+        }
```

Comment:
```suggestion
        if (driverId == null) {
            resp.sendRedirect(req.getContextPath() + "/login");
            return;
        }
```
not sure that this statement needed here, cause if driver is null, then anyway he won't have access to page until he register , but if this works for you,  then before login add /